scripts/package

```diff
@@ -62,10 +62,10 @@ if [ ! -d build/charts ]; then
     done
 
     # Remove index to force building a virtual index like system charts
-    rm -f $INDEX_PATH $CHART_REPO_DIR/assets/index.yaml
+    rm -f $INDEX_PATH $CHART_REPO_DIR/assets/index.yaml $CHART_REPO_DIR/released/assets/index.yaml
 fi
 
-TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go $SYSTEM_CHART_REPO_DIR $CHART_REPO_DIR/assets $IMAGE $AGENT_IMAGE $RUNTIME_IMAGE
+TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go $SYSTEM_CHART_REPO_DIR $CHART_REPO_DIR $IMAGE $AGENT_IMAGE $RUNTIME_IMAGE
```

Comment:
Do the scripts automatically remove duplicate image/tag pairs that get produced from the final list? If we are iterating through charts and assets, everything will appear twice by definition

Reply:
nvm I just saw your output in https://github.com/rancher/rancher/pull/32100#issuecomment-819016174